protostar/utils/cache.py

```diff
@@ -0,0 +1,37 @@
+import os
```

Comment:
I'd move this file to `protostar.self` component. We are going to remove protostar.utils in the future.

Reply:
[f17a8d4](https://github.com/software-mansion/protostar/pull/895/commits/f17a8d44b4ac90c20db69f3a5a3a11d9b0e4224a)